air/src/air/trace_info.rs

```diff
@@ -177,7 +180,7 @@ impl TraceInfo {
 
     /// Returns the number of columns in the auxiliary segment of an execution trace.
     pub fn aux_segment_width(&self) -> usize {
-        self.aux_segment_width
+        self.aux_segment_width + 2 * self.logup_gkr as usize
     }
```

Comment:
I would mention in the doc comment that when LogUp-GKR is enabled, two extra columns are added to the auxiliary trace.

Also, should we rename this to `aux_trace_width()`? To keep it parallel with `main_trace_width()`.

Reply:
I think it is easier to change `main_trace_width` to `main_segment_width` given the rest of the code-base. I think the rational was that there is only one execution trace but, potentially, two segments; a main and an auxiliary one.